src/chttpd/src/chttpd_stats.erl

```diff
@@ -101,7 +111,41 @@ incr(Idx, Count) ->
         #st{} = St ->
             Total = element(Idx, St) + Count,
             NewSt = setelement(Idx, St, Total),
-            put(?KEY, NewSt);
+            put(?KEY, NewSt),
+            maybe_report_intermittent(St);
```

Comment:
This would break accuracy for all request types which are not `_changes`.

Reply:
Good point as this would alter the current behavior a bit where for some requests we would report intermittently when before they didn't.  I think we could change the Mod:report(...) API a bit and make reset conditional only if Mod:report(...) return `true` if it is `false`, then we don't reset and keep accumulating stats.

That way  reporter module could chose to return `false` for example for `_all_docs` request and they'd only get a  final stats report at the end of the request (when HttpResp != undefined). And other endpoints like `_changes` could to periodic reporting and return `true`